modules/Bio/EnsEMBL/Analysis/Hive/Config/RepeatMasking.pm

```diff
@@ -0,0 +1,549 @@
+    faidx_softmasked_genome_file  => catfile($self->o('genome_dumps'), $self->o('species_name').'_softmasked_toplevel.fa.reheader'),
+
+    full_repbase_logic_name => "repeatmask_repbase_".$self->o('repbase_logic_name'),
+
+    'min_toplevel_slice_length'   => 250,
+
+    'repeatmodeler_logic_name'    => 'repeatmask_repeatmodeler',
+    
+    ensembl_analysis_script           => catdir($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts'),
+    sequence_dump_script              => catfile($self->o('ensembl_analysis_script'), 'sequence_dump.pl'),
+    
+    ensembl_misc_script        => catdir($self->o('enscode_root_dir'), 'ensembl', 'misc-scripts'),
+    repeat_types_script        => catfile($self->o('ensembl_misc_script'), 'repeats', 'repeat-types.pl'),
+    
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+    mysql_dump_options => '--max_allowed_packet=1000MB',
+
+########################
+# Executable paths
+########################
+    
+    'blast_type' => 'ncbi', # It can be 'ncbi', 'wu', or 'legacy_ncbi'
+    'dust_path' => catfile($self->o('binary_base'), 'dustmasker'),
+    'trf_path' => catfile($self->o('binary_base'), 'trf'),
+    'repeatmasker_path' => catfile($self->o('binary_base'), 'RepeatMasker'),
+    samtools_path => catfile($self->o('binary_base'), 'samtools'), #You may need to specify the full path to the samtools binary
+    
+########################
+# Misc setup info
+########################
+    'repeatmasker_engine'       => 'crossmatch',
+    'masking_timer_long'        => '5h',
+    'masking_timer_short'       => '2h',
+
+
+########################
+# db info
+########################
+    'reference_db' => {
+      -dbname => $self->o('reference_db_name'),
+      -host   => $self->o('reference_db_server'),
+      -port   => $self->o('reference_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+  };
+}
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  # set the logic names for repeat masking
+  my $wide_repeat_logic_names;
+  if ($self->o('use_repeatmodeler_to_mask')) {
+    $wide_repeat_logic_names = [$self->o('full_repbase_logic_name'),$self->o('repeatmodeler_logic_name'),'dust'];
+  } elsif ($self->o('replace_repbase_with_red_to_mask')) {
+    $wide_repeat_logic_names = [$self->o('red_logic_name'),'dust'];
+  } else {
+    $wide_repeat_logic_names = [$self->o('full_repbase_logic_name'),'dust'];
+  }
+
+  return {
+    %{$self->SUPER::pipeline_wide_parameters},
+    skip_repeatmodeler => $self->o('skip_repeatmodeler'),
+    wide_repeat_logic_names => $wide_repeat_logic_names,
+    wide_ensembl_release => $self->o('ensembl_release'),
```

Comment:
It looks like wide_ensembl_release is not needed anywhere, it can be removed

Reply:
Changes effected